python-backend/app/api/users/minespace/resources/minespace_user_mine.py

```diff
@@ -11,7 +11,7 @@
 
 
 class MinespaceUserMineResource(Resource, UserMixin, ErrorMixin):
-    parser = reqparse.RequestParser()
+    parser = reqparse.RequestParser(trim=True)
```

Comment:
Possibly pedantic, but I think all of these `trim=True` resource changes would have made a nice, separate PR. Then any discussion around it wouldn't need to be mixed in with incidents, and the incidents PR would be considerably smaller

Reply:
Fair point, that also would have made it more visible. Differentiating based on trailing or leading whitespace seems like a bad idea, but I didn't exactly have a conversation. We can have one though.

Reply:
I think it's a good change to have made. My only point was it might be easier to keep an eye on something fairly global like this if it has its own PR. For the future 😄

Marking as resolved